jmix-ui/ui-widgets/src/main/java/io/jmix/ui/widget/client/jqueryfileupload/JmixFileUploadProgressWindow.java

```diff
@@ -427,6 +447,56 @@ public void setProgress(float state) {
         progressBar.setState(state);
     }
 
+    public void updateTotalProgress(int filesLeft) {
+        if (!isTotalProgressVisible()) {
+            return;
+        }
+
+        int currentFileNumber = filesNumber - filesLeft;
+        totalFilesLabel.setText(getTotalProgressDisplayMessage(currentFileNumber));
+
+        float state = (float) currentFileNumber / (float) filesNumber;
+        totalProgressBar.setState(state);
+    }
+
+    protected boolean isTotalProgressVisible() {
+        return filesNumber > 1 && totalProgressDisplayEnabled;
```

Comment:
Let's check `totalProgressEnabled`, so we can avoid checking `filesNumber`, because SingleUpload is used more frequently.